### DISCUSSION TITLE
Pass env var to entry.client.tsx

### DISCUSSION THREAD

**Discussion by cill-i-am** [remix-run/remix#3723] [Q&A] [answered]
I'm trying to implement the new @sentry/remix package and it calls for adding the below to the `entry.client.tsx`

```TS
  Sentry.init({
      dsn: 'https://49a08cb6d7bd459185bd7d5261c6de77@o368240.ingest.sentry.io/6570573',
      tracesSampleRate: 1,
      integrations: [
          new Sentry.BrowserTracing({
              routingInstrumentation: Sentry.remixRouterInstrumentation(
                  useEffect,
                  useLocation,
                  useMatches,
              ),
          }),
      ],
      // ...
  });
```

My problem is that I have the dsn stored in my .env file, which I am happily passing the the `Sentry.init` function in the `entry.server.tsx`. So my question is what's the best way to pass the env var down to this? I know if it was a normal case i'd pass it in the loader. But don't think I can do that here

**Comment by sergiodxa** [comment, marked as answer]
This is documented on a guide here https://remix.run/docs/en/v1/guides/envvars#browser-environment-variables
